features/uri-templating.feature

```diff
@@ -65,6 +66,15 @@ Functionaliteit: URI templating
     | href                                            | templated |
     | /kadasternietnatuurlijkpersonen/{identificatie} | true      |
 
+  Scenario: Verwijzingen naar een Resource type via meerdere properties van één of meerdere gegevensgroepen
+    Gegeven een ZakelijkGerechtigde heeft een verwijzing via de 'stukIdentificaties' property van gegevensgroep 'ZakelijkRecht' naar Stukken
```

Comment:
de properties in de response beginnen niet met een hoofdletter. Dus 'zakelijkRecht' i.p.v. 'ZakelijkRecht'. De schemacomponent (en daarmee denk ik de class begint wel met een hoofdletter. Is het hier handig om de naam uit de component/class te gebruiken i.p.v. de naam van het property in de response?